lib/stripe/resources/customer.rb

```diff
@@ -94,5 +91,32 @@ def self.search(params = {}, opts = {})
     def self.search_auto_paging_each(params = {}, opts = {}, &blk)
       search(params, opts).auto_paging_each(&blk)
     end
+
+    def self.retrieve_cash_balance(customer, _unused = nil, opts = {})
```

Comment:
Can we optionally treat the second parameter as "opts" if no third parameter is specified?

Reply:
Done. We can only really do this for retrieve. We can't do this for update since we can't differentiate between these two cases:

1. `update_cash_balance(customer, nil, params)` using the legacy pattern
2. `update_cash_balance(customer, nil, opts)` using a modern pattern of `(customer_id, params, opts)`

It's not ideal but at least we can give a more informative message to users.